app/frontend/styles/components/search_panel.scss

```diff
@@ -45,7 +43,6 @@
 
   .location-radius-select {
     display: inline-block;
-    float: right;
```

Comment:
We need this to look like this 
![image](https://user-images.githubusercontent.com/25187547/91145058-c8738a00-e6ac-11ea-9d00-e81fb44caa1f.png) and not like this 
![image](https://user-images.githubusercontent.com/25187547/91145079-d4f7e280-e6ac-11ea-85d9-dba1d645d628.png)

Reply:
ah ok, i forgot this appeared, but floating is not the best way to do it and IE might have a problem with it but i havent got ie vm at the moment so will investigate when ive cleared some space on my hard drive